Makefile

```diff
@@ -212,6 +213,14 @@ $(FLAKE8): $(PYTEST)
 test-deps: $(PY)
 	$(call PIP,test-requirements.txt)
 
+$(SELENIUM): $(PY)
+	@# Because shelltoolbox requires ez_setup already installed before being
+	@# installed we need to manually do them this way instead of via the
+	@# requirements.txt.
+	bin/pip install --no-index --no-dependencies --find-links $(WHEEL_CACHE) --find-links $(PYTHON_CACHE) ez_setup==0.9
+	bin/pip install --no-index --no-dependencies --find-links $(WHEEL_CACHE) --find-links $(PYTHON_CACHE) shelltoolbox==0.2.1
```

Comment:
You could make this a callable, so that you don't have to duplicate the --find-links etc. e.g. `$(call PIP_INSTALL, ez_setup==0.9)` or you might even update the current pip calls so you would have something like: `$(call PIP, ez_setup==0.9)`  and `$(call PIP,"-r test-requirements.txt")` (not sure if that would work).